content/applications/finance/fiscal_localizations/vietnam.rst

```diff
@@ -2,20 +2,201 @@
+     information.
+   - In the :guilabel:`Street 2` field, enter the neighborhood.
+
+- :guilabel:`Tax ID`: The tax identificaiton number
+
+.. _vietnam/sinvoice:
+
+E-invoice integration with S-Invoice
+====================================
+
+S-Invoice_  is an e-invoice service platform provided by Viettel, one of the biggest e-invocie service provider in Vietnam.
+Odoo supports integration with S-invoice to submit the invoices generated in Odoo.
+
+.. note::
+   The :guilabel:`Vietnam - E-invoicing module` must be installed to submit invoices to S-invoice.
+
+.. _vietnam/sinvoice/setup:
+
+Set-up
+------
+
+.. _vietnam/sinvoice/setup/registration:
+
+S-Invoice Registration
+~~~~~~~~~~~~~~~~~~~~~~
+
+To send electronic invoices to S-Invoice, you first need to register to the S-Invoice portal to confirm the package, choose
+the invoice template and issue a legal notice for the chosen template.
+
+#. Go to S-Invoice_ and :guilabel:`Log in` on the portal.
+
+.. note::
+   To create an account, contact :guilabel:`S-Invoice` directly on the portal.
+
+#. Follow the instruction of S-Invoice to choose the invoice template and issue a legal notice of the template chosen.
+
+.. _vietnam/sinvoice/setup/odoo:
+
+Configuration in Odoo
+~~~~~~~~~~~~~~~~~~~~~
+
+To connect Odoo database to S-invoice, :guilabel:`Credentials` and :guilabel:`Template invoice` need to be
+set first.
+
+#. Go to :menuselection:`Accounting --> Configuration --> Settings`. Under :guilabel:`Vietnamese Integration`,
+   connect Odoo database with S-Invoice account using :guilabel:`Username` and :guilabel:`Password`. Set a
+   :guilabel:`Default symbol` if needed.
+
+#. Add :guilabel:`S-Invoice Templates` by going to :menuselection:`Accounting --> Configuration --> Templates`.
+
+#. Click :guilabel:`New` to create a new template and add :guilabel:`Template code` and :guilabel:`Template Invoice Type`.
+
+   .. important::
+      For the :guilabel:`Template code`, input the code used by Sinvoice excluding any other information in the template name.
+
+      .. example::
+         If the invoice template is named `1/001 - Hóa đơn GTGT - ND123`, input `1/001` for :guilabel:`Template code`.
+
+
+#. Add :guilabel:`Ivnoice Symbols` by selection. New symbol is available for creation directly or under
+   :menuselection:`Accounting --> Configuration --> Symbol`.
+
+   .. important::
+      Make sure that the :guilabel:`Address` and :guilabel:`Tax ID` of the company contact and the customer's contact is
+      filled in properly.
+
+.. _vietnam/sinvoice/workflow:
+
+Workflow
+--------
+
+.. _vietnam/sinvoice/workflow/sending:
+
+Create an invoice
+~~~~~~~~~~~~~~~~~
+
+Invoices can be sent to MyInvois once they have been confirmed. To do so, follow the
+:ref:`invoice sending <accounting/invoice/sending>` steps, and in the :guilabel:`Send` window,
+enable the :guilabel:`Send to Sinvoice` option and click :guilabel:`Send & Print`.
```

Comment:
@afma-odoo The exact button is 'Send to Sinvoice'.
![image](https://github.com/user-attachments/assets/2a2a64b6-406c-472f-b03b-506bae56f681)

Since all the buttons and info inside Odoo is displayed as "SInvoice", I will convert all "S-Invoice" and "Sinvoice" to "SInvoice"